OutOfSchool/OutOfSchool.WebApi.Tests/Services/Database/SensitiveWorkshopsServiceTests.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using AutoMapper;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using OutOfSchool.BusinessLogic.Models;
+using OutOfSchool.BusinessLogic.Models.Workshops;
+using OutOfSchool.BusinessLogic.Services;
+using OutOfSchool.BusinessLogic.Services.AverageRatings;
+using OutOfSchool.BusinessLogic.Services.Images;
+using OutOfSchool.BusinessLogic.Services.Workshops;
+using OutOfSchool.Services.Enums;
+using OutOfSchool.Services.Models;
+using OutOfSchool.Services.Models.ChatWorkshop;
+using OutOfSchool.Services.Repository;
+using OutOfSchool.Tests.Common;
+using OutOfSchool.Tests.Common.TestDataGenerators;
+
+namespace OutOfSchool.WebApi.Tests.Services.Database;
+
+[TestFixture]
+public class SensitiveWorkshopsServiceTests
+{
+    private readonly string includingPropertiesForMappingDtoModel = $"{nameof(Workshop.Address)},{nameof(Workshop.Teachers)}," +
+        $"{nameof(Workshop.DateTimeRanges)},{nameof(Workshop.InstitutionHierarchy)}";
+
+    private ISensitiveWorkshopsService sensitiveWorkshopService;
+    private Mock<IWorkshopRepository> workshopRepository;
+    private Mock<IEntityRepositorySoftDeleted<long, DateTimeRange>> dateTimeRangeRepository;
+    private Mock<IEntityRepositorySoftDeleted<Guid, ChatRoomWorkshop>> roomRepository;
+    private Mock<ITeacherService> teacherService;
+    private Mock<ILogger<WorkshopService>> logger;
+    private Mock<IMapper> mapperMock;
+    private Mock<IImageDependentEntityImagesInteractionService<Workshop>> workshopImagesMediator;
+    private Mock<IProviderAdminRepository> providerAdminRepository;
+    private Mock<IAverageRatingService> averageRatingServiceMock;
+    private Mock<IProviderRepository> providerRepositoryMock;
+    private Mock<ICurrentUserService> currentUserServiceMock;
+    private Mock<IMinistryAdminService> ministryAdminServiceMock;
+    private Mock<IRegionAdminService> regionAdminServiceMock;
+    private Mock<ICodeficatorService> codeficatorServiceMock;
+    private string userId;
```

Comment:
(Excessive fields) Same comment as in previous test class. Please don't use something as a filed if it could be a local variable.

Reply:
Fixed